install_config/redeploying_certificates.adoc

```diff
@@ -431,11 +431,11 @@ You can set the `openshift_redeploy_service_signer=false` parameter in the inven
 If you set `openshift_redeploy_openshift_ca=true` and `openshift_redeploy_service_signer=true` in the inventory file, the service signing certificate is redeployed when you redeploy the master certificates. If you set `openshift_redeploy_openshift_ca=false` or omit the parameter, the service signer certificate is never redeployed.
 
 [[redeploying-named-certificates]]
-=== Redeploying Only Named Certificates
+=== Redeploying Named and Web console Certificates
 
-The *_openshift-master/redeploy-named-certificates.yml_* playbook redeploys only named certificates. Running this playbook also completes serial restarts of master services.
+The *_openshift-master/redeploy-named-certificates.yml_* playbook redeploys named certificates and web console certificates. Running this playbook also completes serial restarts of master services.
 
-To redeploy named certificates only, change to the directory that contains the playbooks, and run this playbook.
+To redeploy named and web console certificates, change to the directory that contains the playbooks, and run this playbook.
```

Comment:
It's nitpicky, but because line 436 uses "named certificates and web console certificates", I'd follow the tedium of "To redeploy named certificates and web console certificates,..."

Not crucial though.